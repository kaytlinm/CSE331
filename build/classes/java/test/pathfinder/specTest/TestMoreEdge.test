#Tests that returns path with least cost when other paths with more edges and greater cost
CreateGraph graph1

AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

AddEdge graph1 n1 n2 2.0
AddEdge graph1 n2 n3 1.0
AddEdge graph1 n1 n3 4.0

FindPath graph1 n1 n3
