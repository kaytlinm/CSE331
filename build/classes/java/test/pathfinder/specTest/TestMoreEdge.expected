#Tests that returns path with least cost when other paths with more edges and greater cost
created graph graph1

added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

added edge 2.000 from n1 to n2 in graph1
added edge 1.000 from n2 to n3 in graph1
added edge 4.000 from n1 to n3 in graph1

path from n1 to n3:
n1 to n2 with weight 2.000
n2 to n3 with weight 1.000
total cost: 3.000